lib/liquid/standardfilters.rb

```diff
@@ -198,6 +214,8 @@ def map(input, property)
           r.is_a?(Proc) ? r.call : r
         end
       end
+    rescue TypeError
+      raise_property_error(property)
```

Comment:
In my previous iterations I stored the InputIterator instance in a variable to isolate the begin/rescue block from the initialization in order to only catch `TypeError` from the mapping functionality. However, after looking at the InputIterator `initialize` method, there doesn't seem to be a cause of TypeError to being raised from a user writing in Liquid.